client/components/social-buttons/google.js

```diff
@@ -117,12 +121,10 @@ class GoogleLoginButton extends Component {
 
 		const { responseHandler } = this.props;
 
-		// Handle click async if the library is not loaded yet
-		// the popup might be blocked by the browser in that case
-		// options are documented here:
+		// Options are documented here:
 		// https://developers.google.com/api-client-library/javascript/reference/referencedocs#gapiauth2signinoptions
-		this.initialize()
-			.then( gapi => gapi.auth2.getAuthInstance().signIn( { prompt: 'select_account' } ).then( responseHandler ) )
+		Promise.resolve( window.gapi.auth2.getAuthInstance().signIn( { prompt: 'select_account' } ) )
```

Comment:
Huh, weird. It didn't work for me earlier, but that may've been something else I broke.